### PR TITLE
Cloudshellstorage

### PR DESCRIPTION
**Pull request by maertendMSFT**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: 101-cloud-shell-vnet-storage/prereq.azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
+{
```
Comment (BellaLi) [inline 101-cloud-shell-vnet-storage/prereq.azuredeploy.json:1]: Please move to prereqs folder.

File: 101-cloud-shell-vnet-storage/prereq.azuredeploy.parameters.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (BellaLi) [inline 101-cloud-shell-vnet-storage/prereq.azuredeploy.parameters.json:1]: Please move to prereqs folder.

File: 101-cloud-shell-vnet-storage/settings.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (BellaLi) [inline 101-cloud-shell-vnet-storage/settings.json:1]: Please move to prereqs folder.

File: 101-cloud-shell-vnet-storage/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Sample name
```
Comment (bmoore-msft) [inline 101-cloud-shell-vnet-storage/README.md:1]: check the other PR for comments and apply the same here

**Comment by bmoore-msft** [comment]
@BellaLi -  Not sure how, but there are merge conflicts in this even though I don't see the file changes...  I think you might have to submit this one again from a clean branch...

**Comment by maertendMSFT** [comment]
@bmoore-msft I will do that right now if you are merging :)

**Comment by maertendMSFT** [comment]
@bmoore-msft new PR is here: https://github.com/Azure/azure-quickstart-templates/pull/7817

**Comment by bmoore-msft** [comment]
merged via #7817

**Pull request closed without merging**